src/Views/FirmwareView.vala

```diff
@@ -46,8 +46,7 @@ public class About.FirmwareView : Granite.SimpleSettingsPage {
         progress_alert_view.get_style_context ().remove_class (Gtk.STYLE_CLASS_VIEW);
 
         placeholder_alert_view = new Granite.Widgets.AlertView (
-            _("Checking for Updates"),
-            _("Connecting to the firmware service and searching for updates."),
```

Comment:
Is it OK to just drop this?

Reply:
No, the alertview needs to have two lines or it won't compile

Reply:
And the second can't be empty even?

Reply:
It's not designed to be empty, it would not be spaced or aligned correctly

Reply:
Nice to know :)

Reply:
In US English it will display as "1", which is intended for that locale